src/PDPVerifier.sol

```diff
@@ -310,18 +311,11 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
         require(proofSetLive(setId), "Proof set not live");
         require(rootData.length > 0, "Must add at least one root");
         require(proofSetOwner[setId] == msg.sender, "Only the owner can add roots");
-        bool needsChallengeEpoch = nextChallengeEpoch[setId] == 0;
         uint256 firstAdded = nextRootId[setId];
 
         for (uint256 i = 0; i < rootData.length; i++) {
             addOneRoot(setId, i, rootData[i].root, rootData[i].rawSize);
         }
-        // Initialise the first challenge epoch and challengeable leaf range when the first data is added.
```

Comment:
Would you be good to document on `addRoots` that a prover HAS to call `nextProvingPeriod` before these trees are eligible for proving.